.eslintrc.json

```diff
@@ -12,7 +12,7 @@
     "ecmaVersion": 2018,
     "sourceType": "module"
   },
-  "extends": ["airbnb", "plugin:react/recommended", "plugin:react-hooks/recommended"],
+  "extends": ["plugin:react/recommended", "plugin:react-hooks/recommended"],
   "plugins": ["react"],
   "rules": {
```

Comment:
- [ ] In your components `Button.js` and `Display.js`, you have disabled linters and this is seriously frowned upon in Microverse and professional workplaces.  Linters play a crucial role in code quality and maintainability. Disabling linters, especially without a valid reason, is generally discouraged. Here are a few reasons why linters should not be disabled:

    - Code Consistency: Linters enforce a set of coding style guidelines, ensuring consistent formatting and coding practices across the codebase. Consistent code is easier to read, understand, and maintain, especially when multiple developers are working on the same project.
    
    - Readability: Linters help improve code readability by highlighting potential issues, such as unused variables, missing semicolons, or inconsistent indentation. By addressing these issues, code becomes more comprehensible and less prone to bugs or misunderstandings.
    
    - Code Quality: Linters can detect common programming errors and anti-patterns, such as unused variables, unreachable code, or inappropriate variable naming. By catching these issues early on, linters help improve code quality, reducing the likelihood of bugs and improving overall code reliability.
    
    - Best Practices: Linters often incorporate best practices and coding standards from the community, language, or framework. Following these best practices helps ensure that code is written in a way that aligns with industry standards, leading to more maintainable and robust software.
    
    - Collaboration: When multiple developers are working on a project, linters act as a common set of guidelines, making it easier for everyone to understand and work with each other's code. Linters facilitate smoother collaboration and reduce the cognitive load of understanding varying coding styles.
    
    - Continuous Integration: Many modern development practices include automated build and integration processes. Linters are often integrated into these workflows, ensuring that code adheres to standards before being merged into the main codebase. Disabling linters could result in build failures or inconsistent code quality checks.

_Please remove all the linter disable commands from you code base. fix the linter issues and push your code._